app/models/book.rb

```diff
@@ -0,0 +1,8 @@
+class Book < ApplicationRecord::Base
+  validates :genere, presence: true
```

Comment:
If you are applying the same validation to several fields, you can group all of them in one line  
`validates :genere, :author, :image, :title,  :publisher, :year, presence: true`